src/NodeParser/TypeReferenceNodeParser.ts

```diff
@@ -45,7 +46,7 @@ export class TypeReferenceNodeParser implements SubNodeParser {
         }
 
         if (typeSymbol.flags & ts.SymbolFlags.TypeParameter) {
-            return context.getArgument(typeSymbol.name);
+            return context.getArgument(typeSymbol.name) || new UnknownType(true);
```

Comment:
```suggestion
            return context.getArgument(typeSymbol.name) ?? new UnknownType(true);
```

Just to be more explicit.